web/widgets.py

```
@@ -135,3 +135,268 @@ def __init__(self, attrs=None):
         if attrs:
             default_attrs.update(attrs)
         super().__init__(default_attrs)
+
+
+class TailwindJSONWidget(forms.Widget):
+    template_name = "admin/widgets/json_features_widget.html"
+
+    def __init__(self, attrs=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)